frontend/src/reducers/viewHandler.js

```diff
@@ -399,6 +377,23 @@ export default function viewHandler(state = initialState, action) {
       };
     }
 
+    case TOGGLE_INCLUDED_VIEW: {
+      const { id, showIncludedView } = action.payload;
+      const view = getLocalView(state, id);
+
+      return {
+        ...state,
+        views: {
+          ...state.views,
+          [`${id}`]: {
+            ...view,
+            isShowIncluded: !!showIncludedView,
```

Comment:
why do we have isShowIncluded and also hasShowIncluded with the same value. Couldn't we just use one of them ?

Reply:
I have no idea. Looks like some hack from the old days. I did not wanted to touch it, since it's not in the Table component but looks to be safe.